EIPS/eip-1057.md

```diff
@@ -570,6 +570,6 @@ The reference ProgPoW mining implementation is located at [the @ifdefelse ProgPO
 
 ## License and Copyright
 
-The ProgPoW algorithm and this specification are a new work.  Copyright and related rights are waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
+The ProgPoW algorithm and this specification are a new work.  Copyright and related rights are waived via [CC0](../CC0).
```

Comment:
If we are going to fix all of the copyright lines, we should fix this one while we are at it.  Section header and copyright line should be standardized.